pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -342,12 +380,12 @@ func modifyOutboundRouteConfig(push *model.PushContext, in *plugin.InputParams,
 	return httpRoute
 }
 
-func buildInboundRouteConfig(push *model.PushContext, in *plugin.InputParams, instance *model.ServiceInstance) *mccpb.ServiceConfig {
-	config := in.Env.IstioConfigStore
+func buildInboundRouteConfig(push *model.PushContext, env *model.Environment, instance *model.ServiceInstance) *mccpb.ServiceConfig {
+	config := env.IstioConfigStore
 
 	attrs := addDestinationServiceAttributes(make(attributes), push, instance.Service.Hostname)
 	out := &mccpb.ServiceConfig{
-		DisableCheckCalls: disablePolicyChecks(inbound, in.Env.Mesh, in.Node),
+		DisableCheckCalls: env.Mesh.DisablePolicyChecks,
```

Comment:
This is clearly not right. Are you sure you have rebased correctly?

Reply:
Fixed